src/resources/events/EventsRoutes.ts

```diff
@@ -14,18 +14,21 @@ import { SetEventOrganizerRequest } from "../../generated/models/SetEventOrganiz
 import { UpdateEventRequest } from "../../generated/models/UpdateEventRequest.generated";
 import { getPagination } from "../../utils/RequestUtil";
 import { EventsController } from "./controllers/EventsController";
+import { Permit } from "../auth/middleware/Permit";
 
 const log: debug.IDebugger = debug("app:events-routes");
 
 @Service()
 export class EventsRoutes {
 	constructor(public eventsController: EventsController) {}
 
+	static readonly basePath: string = "/events";
```

Comment:
Bei solchen Konstanten würde ich den Type einfach implizit lassen, weil dann weniger Code nötig ist. Ist aber auch so fein!

```suggestion
	static readonly basePath = "/events";
```